README.md

```diff
@@ -1,2 +1,37 @@
 # credentials-io
 Library for reading/writing credentials
+
+## API
+
+### verify(identity, options, callback)
+Verifies digital signature on identity object and any credentials it contains.
+The callback recieves the following object:
+```
+{
+ verified: <true/false>,
+ identity: <framed identity>,
+ domain: <domain the signature was signed for>,
+ publicKey: <public key object for the creator of the signature>,
+ publicKeyOwner: <identity object for the creator of the signature>,
+ // a map of verified objects for each credential (ID => verified)
+ credentials: {}
+}
+```
+
+### send(options, callback)
```

Comment:
These parameters need to be documented. Look at this for an example on how to document parameters: https://github.com/digitalbazaar/credentials-polyfill/blob/master/README.md#registering-a-decentralized-identifier